pkg/orchestration/wiring/testdata/e2e.tests.bundle.yaml

```diff
@@ -575,7 +575,9 @@ spec:
           secretName: ec2---iamrole
   - name: ec2--instance
     references:
-    - resource: ec2--secretenvvar
+    - name: ec2--secretenvvar-metadata-name
+      path: metadata.name
+      resource: ec2--secretenvvar
```

Comment:
Would be great to ensure we set the `example` field in ALL the references we generate in a follow up PR. I think there is a ticket to do that...